util/jsonrpc-types/src/pool.rs

```diff
@@ -70,13 +70,13 @@ impl From<CorePoolTransactionEntry> for PoolTransactionEntry {
 #[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Hash, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum OutputsValidator {
-    /// "default": The default validator which restricts the lock script and type script usage.
+    /// "well_known_scripts_only": The default validator which restricts the lock script and type script usage.
     ///
-    /// The default validator only allows outputs (a.k.a., cells) that
+    /// This validator only allows outputs (a.k.a., cells) that
     ///
-    /// * use either the secp256k1 or the secp256k1 multisig bundled in the genesis block via type script hash as the lock script,
-    /// * and the type script is either empty or DAO.
-    Default,
+    /// * use `code_hash` and `hash_type` listed in https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md as the lock or type script,
+    /// * use `code_hash` and `hash_type` listed in `custom_well_known_scripts` of `ckb.toml`
+    WellKnownScriptsOnly,
```

Comment:
[The generated README](https://github.com/nervosnetwork/ckb/pull/2821/files#diff-6fe1f4e31ab0e32a0eb34735313912f3bf54073b53bb7d67b9af1f41d3f89c44R3788-R3790) does not include this part. Is it a bug?

Reply:
it looks like an existing bug of `rpc.py` which cannot generate multiple line text with enum variant, for example, compare the doc of `DepType`:
https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md#type-deptype
https://github.com/nervosnetwork/ckb/blob/develop/util/jsonrpc-types/src/blockchain.rs#L276-L278

let me try to fix it in a new PR and re-generate the readme file for all variants.